src/controllers/request.js

```diff
@@ -95,4 +99,80 @@ export default class usersController {
     });
     }
   }
+
+  static async filterTrips(req, res) {
+
+try {   
+   if (req.query.filterKey) {
+      const { filterKey} = req.query;
+      const searchQuery = [
+        where(
+          cast(col('id'), 'varchar'),
+          { [Op.like]: `%${filterKey}%` }
+        ),
+        where(
+          cast(col('requesterId'), 'varchar'),
+          { [Op.like]: `%${filterKey}%` }
+        ),
+        where(
+          cast(col('type'), 'varchar'),
+          { [Op.like]: `%${filterKey}%` }
+        ),
+        where(
+          cast(col('status'), 'varchar'),
+          { [Op.like]: `%${filterKey}%` }
+        ),
+        { reason: { [Op.like]: `%${filterKey}%`, }, },
+        { origin: { [Op.like]: `%${filterKey}%`, }, },
+        { destination: { [Op.like]: `%${filterKey}%`, }, },        
+      ]; 
+      
```

Comment:
@Niyonsengaeric  please use the case insensitive feature for the best user experience. The user should be able to get results when they used either lower case or upper case search queries.

Reply:
Done!

Reply:
Good job @Niyonsengaeric .